nsxt/resource_nsxt_policy_tier0_gateway.go

```diff
@@ -674,12 +686,14 @@ func resourceNsxtPolicyTier0GatewayBGPConfigSchemaToStruct(cfg interface{}, isVr
 			log.Printf("[WARNING] BGP graceful restart settings are not applicable for VRF gateway %s, and will be ignored", gwID)
 		}
 	} else {
-		routeStruct.InterSrIbgp = &interSrIbgp
-		routeStruct.LocalAsNum = &localAsNum
```

Comment:
Are you intentionally removing this line? (maybe it's not necessary)

Reply:
Yes, this attribute is assigned in line 681